### ISSUE TITLE
Component library styles don't load

### ISSUE THREAD

**Issue by danielbayley80** [skeletonlabs/skeleton#1753]
### Current Behavior

Possibly a limit of my noob understanding, but I  have created a library project for UI components based on SkeltonUI with a peer dependency.

When I include them in my project it appears that the classes used in the component are not being loaded.

When I then include the classes in the projects page, they then load and the component renders as expected.

I don't really understand how the classes are being recognised and built, but it appears they are not if they are used in a component library. 

### Expected Behavior

Assuming it is not an inherent limitation of the library, I would expect all classes being used to be loaded including those from a library which is used in the project.

### Steps To Reproduce

Create a simple UI Component library.
Add a text input field to a component.
Add the material theme to it (this one is more obvious to see its not working).
Create a web app project with skeleton UI.
Reference the UI component.
Add the component to page.
Load the page.
You should see the text box is rounded.
Now add a text box to the same page alongside the UI component.
Add the Material style to it.
Both text boxes will now display using the correct material UI.

### More Information

I'm pretty new to Svelte and SkeletonUI and I dont really understand how the styles are being compiled dynamically. This might be entirely a limit of my understanding.

**Comment by Sarenor** [comment]
Not a bug and even if it would be, it'd be a bug in Tailwind, at least if I understood your issue correctly.

The CSS Classes are build with Tailwind and Tailwinds compiler. The compiler checks your project files for the classes you use and only includes used classes in the css bundle.
Since the classes are in a library and therefore out of the scanning scope, the library classes won't be included.
Luckily, Skeleton has the same problem and there's an easy fix for it.
In your ``tailwind.config.cjs`` there's already an exception for Skeleton, you just need to add one for your project as well:
```js
const config = {
    content: [
        './src/**/*.{html,js,svelte,ts}',
        './node_modules/@skeletonlabs/skeleton/**/*.{html,js,svelte,ts}',
        // Add the path to your library here with the regex for filetypes that should be scanned
    ],

    theme: {
        extend: {}
    },
    darkMode: 'class',
    plugins: [
        require('@tailwindcss/forms'),
        ...require("@skeletonlabs/skeleton/tailwind/skeleton.cjs")()
    ]
};

module.exports = config;
```

**Comment by danielbayley80** [comment]
Thanks. I think the classes I'm using are skeleton as well, but the same applies.
 > The compiler checks your project files for the classes you use and only includes used classes in the css bundle.
 
 This is the key issue. Granted it's more of a tailwind issue, although the first time I used Tailwind was when I started using Skeleton. I went through the manual steps and followed the tailwind.config.cjs instructions, but didn't really appreciate what it was doing. It might be worth a mention on the setup page or a sentence as to why, basically what you just said to me ....
 
 "The compiler checks your project files for the classes you use and only includes used classes in the css bundle. you'll need to add Skeleton and any other component libraries to tailwind.config.cjs" - that would be enough to jog a noob like me.
 
 Thanks for your help.
